etc/config/compiler-explorer.defaults.properties

```diff
@@ -12,6 +12,7 @@ optionsBlacklistRe=^(-W[alp],)?((--?(wrapper|fplugin.*|specs|load|plugin|include
 allowedShortUrlHostRe=^([-a-z.]+\.)?(xania|godbolt)\.org$
 googleShortLinkRewrite=^https?://goo.gl/(.*)$|https://godbolt.org/g/$1
 urlShortenService=none
+truncateErrorMessages=50
```

Comment:
Should it be 50 by default? I think having it **not** truncating for self hosted CE instances is a better alternative, and then having some limit only for the Amazon instances

Reply:
That makes sense.

Is there a way to test the configuration for the amazon instances locally?

Reply:
Yes! create a `compiler-explorer.local.properties` which will be top priority file, and then just put those changes on the amazon version once they seem to work :)